### DISCUSSION TITLE
Support for `pkl` config files

### DISCUSSION THREAD

**Discussion by henriqueleite42** [eslint/eslint#18631] [Ideas]
[Apple Pickle](https://github.com/apple/pkl) is created for improving configuration files, would be amazing to be able to use it in ESLint.

**Comment by nzakas** [comment]
Thanks for the idea. Given that we just switched to a new configuration file format, and that we need JavaScript to be active in those configuration files, it's unlikely that ESLint will be changing how we do configuration anytime soon.
